src/crawlee/storages/request_queue.py

```diff
@@ -97,7 +96,7 @@ def __init__(
         self._internal_timeout_seconds = 5 * 60
         self._assumed_total_count = 0
         self._assumed_handled_count = 0
-        self._queue_head_dict: OrderedDictType[str, str] = OrderedDict()
+        self._queue_head_dict: OrderedDict[str, str] = OrderedDict()
```

Comment:
so this line is the actual fix? it feels like you only changed the types, i thought that cannot have a runtime effect (just like with TS)?

or is it about the removed unknown label above?

Reply:
'tis but a draft... I just fixed a bit of weirdness I noticed in the file. Sadly the actual fix still evades me.